UI/src/router.js

```diff
@@ -315,6 +400,42 @@
         speedLimitCentering('speedLimit', id);
       },
 
+      hazardousMaterialTransportProhibitionErrors: function (id, linkId) {
```

Comment:
`function (id, linkId) {` the parameter name id could be change in this function and the ones after this. It could be change for idType or something like that